json-endpoints.php

```diff
@@ -2189,146 +2189,6 @@
 	'group'       => '__do_not_document',
 	'stat'        => 'invites:1',
 	'method'      => 'POST',
-	'path'        => '/sites/%s/invites/%s',
```

Comment:
I can't seem to find a place where they are duplicated, can you please point me?